src/main/java/codeu/controller/ProfileServlet.java

```diff
@@ -1,18 +1,118 @@
 package codeu.controller;
 
+import codeu.model.data.Conversation;
+import codeu.model.data.Message;
+import codeu.model.data.User;
+import codeu.model.store.basic.ConversationStore;
+import codeu.model.store.basic.MessageStore;
+import codeu.model.store.basic.UserStore;
 import java.io.IOException;
+import java.time.Instant;
+import java.util.List;
+import java.util.UUID;
 import javax.servlet.ServletException;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Whitelist;
 
 
+import org.mindrot.jbcrypt.BCrypt;
 /** Servlet class responsible for the Profile page. */
 public class ProfileServlet extends HttpServlet {
+	private UserStore userStore;
+	
+	private ConversationStore conversationStore;
+	
+	private MessageStore messageStore;
+	
+	@Override
+	  public void init() throws ServletException {
+	    super.init();
+	    setUserStore(UserStore.getInstance());
+	    setConversationStore(ConversationStore.getInstance());
+	    setMessageStore(MessageStore.getInstance());
+	    
+	  }
+
+	  /**
+	   * Sets the UserStore used by this servlet. This function provides a common setup method for use
+	   * by the test framework or the servlet's init() function.
+	   */
+	  void setUserStore(UserStore userStore) {
+	    this.userStore = userStore;
+	  }
+	  
+	  void setConversationStore(ConversationStore conversationStore) {
+		 this.conversationStore = conversationStore;
```

Comment:
this.userStore indent is correct. This one is too far. Also, the method itself (and other methods in the file) are indented too far I think